features/FEATURE_BLE/ble/DiscoveredCharacteristic.h

```diff
@@ -551,8 +552,9 @@ class DiscoveredCharacteristic {
      * @return true if operands are equals and false otherwise.
      */
     friend bool operator==(
-        const DiscoveredCharacteristic& lhs, const DiscoveredCharacteristic& rhs
-    ) {
+        const DiscoveredCharacteristic &lhs, const DiscoveredCharacteristic &rhs
+    )
```

Comment:
Again, I'm not so sure to understand why the opening brace should be on a new line.